lib/iris/tests/integration/test_pp.py

```diff
@@ -414,7 +414,7 @@ def test_hybrid_height_round_trip_no_reference(self):
             "Unable to create instance of HybridHeightFactory. "
             "The source data contains no field(s) for 'orography'."
         )
-        warn.assert_called_once_with(msg)
+        warn.assert_called_with(msg)
```

Comment:
I'm concerned that we got this error before you made the change:

```
FAIL: test_hybrid_height_round_trip_no_reference (iris.tests.integration.test_pp.TestVertical)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/tmp/cirrus-ci-build/lib/iris/tests/integration/test_pp.py", line 417, in test_hybrid_height_round_trip_no_reference
    warn.assert_called_once_with(msg)
  File "/tmp/cirrus-ci-build/.nox/tests/lib/python3.7/unittest/mock.py", line 888, in assert_called_once_with
    raise AssertionError(msg)
AssertionError: Expected 'warn' to be called once. Called 2 times.
```

Doesn't this mean behaviour has changed? Why?

Reply:
I'll see if I can replicate this locally...

Reply:
Okay, so frustratingly I can't recreate this failure locally. Either with a test environment that I resolve myself from the `py37` dependencies, nor with the `py37` lock...

I'm not overly concerned about this... although, I'm not loving this inconsistency in behavour. I could be down to a `nose` race condition... dunno.

That said, I'd be more worried if the warning wasn't being generated.

So, personally, I'm not seeing this as a blocker to merging unless you guys want to pursue it further.

Reply:
I think I’ve seen this elsewhere, and it went away when tests were re-spun.